doc/classes/EditorSettings.xml

```diff
@@ -520,6 +605,10 @@
 		<member name="editors/visual_editors/visual_shader/port_preview_size" type="int" setter="" getter="">
 			The size to use for port previews in the visual shader uniforms (toggled by clicking the "eye" icon next to an output). The value is defined in pixels at 100% zoom, and will scale with zoom automatically.
 		</member>
+		<member name="export/ssh/scp" type="String" setter="" getter="">
+		</member>
+		<member name="export/ssh/ssh" type="String" setter="" getter="">
+		</member>
```

Comment:
```suggestion
		<member name="export/ssh/ssh" type="String" setter="" getter="">
			SSH path used for exporting the project to remote device via SSH.
		</member>
```
Not sure if 100% accurate.

Reply:
CC @godotengine/network

Reply:
This is correct. You can leave it empty on most setups, as it'll attempt to use `ssh` from `PATH` if left empty.